examples/datadog-express-apm/host-agent/src/sdk-v3.js

```diff
@@ -16,23 +18,25 @@ const authMiddlewareOptions = {
     },
   },
   scopes: [`manage_project:${projectKey}`],
-  fetch,
+  httpClient: fetch,
 }
 
 const httpMiddlewareOptions = {
   host: 'https://api.europe-west1.gcp.commercetools.com',
   includeRequestInErrorResponse: false,
   includeOriginalRequest: true,
-  fetch,
+  httpClient: fetch,
 }
 
 // datadog options
-const agent = require('path').join(__dirname, '..', 'tracer.js')
 const telemetryOptions = {
   createTelemetryMiddleware,
   userAgent: 'typescript-sdk-middleware-datadog',
-  tracer: require(agent).init(),
-  // apm: () => require('dd-trace').init()
+  tracer: () => tracer.init(),
+  apm: () => require('dd-trace').init(),
```

Comment:
We can remove the `apm: () => require('dd-trace').init()` update the `tracer: () => require('dd-trace').init()` also, since you introduced the lazy module import, can you also update all the `require` statements to use this lazy import.

```suggestion
  tracer: async () => await import('dd-trace').init()
```
Also, I didn't test this, but I believe it should work. Please verify that it actually works.

Reply:
So about your suggestion, I believe it's better to have it as it is so the customers can have the chance to modify tracer.js since this is just and example for them. What do you think?

About `can you also update all the require statements to use this lazy import.`, what do you mean? could you give me an example?

Reply:
I don't think it is necessary to have the tracer.js anymore. if we are importing the `dd-trace` then keeping the `tracer.js` around is redundant. Also, this is the option being passed to the `createTelemetryMiddleware` so it doesn't matter, the customer can change it as they wish, they don't need the `tracer.js` file to be able to make these changes.

Reply:
> About can you also update all the require statements to use this lazy import., what do you mean? could you give me an example?

I think you can leave this out, this might require you to change many other unrelated files, as I can now see it's a commonjs file and changing all the `require` to `import` might make this too much unnecessary work. So just disregard it totally.